osu.Game/Graphics/Cursor/GameplayCursor.cs

```diff
@@ -117,6 +120,16 @@ private void load(OsuConfigManager config)
                 cursorScale = config.GetBindable<double>(OsuConfig.GameplayCursorSize);
                 cursorScale.ValueChanged += newScale => cursorContainer.Scale = new Vector2((float)cursorScale);
                 cursorScale.TriggerChange();
+
+                autoCursorScale = config.GetBindable<bool>(OsuConfig.AutoCursorSize);
+                autoCursorScale.ValueChanged += newScale =>
+                {
+                    if (newScale)
+                        cursorContainer.Scale *= scaling_factor / (game?.Beatmap?.Value.Beatmap.BeatmapInfo.Difficulty.CircleSize ?? scaling_factor);
+                    else
+                        cursorScale.TriggerChange();

```

Comment:
nah you don't want to do this. have all the scale computation logic in one place, not split across `ValueChanged` methods of two bindables. you're going to already be running into weird scenarios if for instance cursorScale is changed when autoCursorScale it enabled.